components/scream/src/physics/p3/atmosphere_microphysics.cpp

```diff
@@ -198,8 +205,8 @@ void P3Microphysics::initialize_impl (const RunType /* run_type */)
   add_postcondition_check<FieldWithinIntervalCheck>(get_field_out("nr"),m_grid,0.0,1.e9,false);
   add_postcondition_check<FieldWithinIntervalCheck>(get_field_out("ni"),m_grid,0.0,1.e9,false);
   add_postcondition_check<FieldWithinIntervalCheck>(get_field_out("bm"),m_grid,0.0,1.0,false);
-  add_postcondition_check<FieldWithinIntervalCheck>(get_field_out("precip_liq_surf"),m_grid,0.0,0.001,false);
-  add_postcondition_check<FieldWithinIntervalCheck>(get_field_out("precip_ice_surf"),m_grid,0.0,0.001,false);
+  add_postcondition_check<FieldWithinIntervalCheck>(get_field_out("precip_liq_surf"),m_grid,0.0,100.0,false);
+  add_postcondition_check<FieldWithinIntervalCheck>(get_field_out("precip_ice_surf"),m_grid,0.0,100.0,false);
```

Comment:
our new precip variables will have magnitude which scales by dt, so it's hard to say how big they should be. To solve this, I'd suggest converting them to the old flux-form precip rates for these condition checks. In other words, replace the line above with `...get_field_out("precip_liq_surf")/dt/C::RHO_LIQ, m_grid,0.0,0.001,false)`

Reply:
Yeah, that's what I thought we may need to do.  I didn't do something scaled by `dt` originally because `dt` is not passed to the initialize_impl, so the information is unknown when the postcondition check is set.  I feel like making it so this is possible may be more work than this PR is set out to do.  Maybe I could suppress these checks for now?  @PeterCaldwell ?

Reply:
Let's instead set the max values to a very big number. I have a vague recollection that "Inf" or something is an allowable value? I think there's a lot of benefit in making sure precip isn't negative even if we don't know how big the max can be.

Reply:
Okay, that just means switch the check to a lower-bound check rather than an interval check.

Reply:
Oh yeah, you could just do that ;-).